phenotrex/util/helpers.py

```diff
@@ -31,3 +31,7 @@ def get_groups(records: List[TrainingRecord]) -> np.ndarray:
     """
     group_ids = np.array([x.group_id for x in records])
     return group_ids
+
+
+def fail_missing_dependency(*args, **kwargs):
+    raise ImportError('FASTA file annotation is not supported - optional dependencies missing.')
```

Comment:
The msg is misleading. "not supported" suggests, that phenotrex is not capable of doing so. Also, the msg should state which dependency is missing, so that users can easily install them. 
I suggest: `"FASTA file annotation requires the optional dependency deepnog. Install with pip."`